bionic/task_state.py

```diff
@@ -339,21 +336,33 @@ def initialize(self, bootstrap, flow_instance_uuid):
             for task_key in self.task_keys
         ]
 
+        can_persist = (
+            single_unique_element(
+                entity_def.can_persist
+                for entity_def in self.entity_defs_by_dnode.values()
+            )
+            and not self.output_would_be_missing()
+        )
+        if can_persist and bootstrap is None:
+            descriptors = [
+                task_key.dnode.to_descriptor() for task_key in self.task_keys
+            ]
+            message = f"""
+            Descriptors {descriptors!r} are set to be persisted but that's
+            impossible; the cache is not available yet because core
+            bootstrap entities depend on it.
+            We will not be saving or reloading these decorators which may
```

Comment:
"The corresponding values will not be serialized and deserialized, which may cause the values to be subtly different."

Note:
- We don't normally refer to Bionic as "We"; we can just use a passive description ("will not be serialized").
- It's not the decorators being saved, it's the values!
- I used "serialized/deserialized" instead of "saved/reloaded" to make it slightly clearer where the difference is coming from.